app/controllers/items_controller.rb

```diff
@@ -1,9 +1,33 @@
-
-
 class ItemsController < ApplicationController
   before_action :authenticate_user!
   # before_action :require_permission, only: [:destroy]
 
+  # def index
+  #   @items = Item.all
+  #   respond_to do |format|
+  #     format.html
+  #     format.json do
+  #       render json: 2
+  #     end
+  #   end
+  # end
+
+  def index
+    @lists = List.all
+    @list = List.new
+    @items = Item.all
+    @item = Item.new
+    respond_to do |format|
+      format.html
+      format.json do
+        categories = @lists.map{ |list| list.category }
```

Comment:
Space missing to the left of {.<br>Pass `&:category` as an argument to `map` instead of a block.